agent/pkg/handler/linkerd_info.go

```diff
@@ -45,6 +45,7 @@ func (h *LinkerdInfo) Start() {
 		select {
 		case <-ticker.C:
 			h.handleCertsInfo(context.Background())
+			h.handleAuthInfo(context.Background())
```

Comment:
maybe have `policy` in the name somewhere? seeing `h.handleAuthInfo` made me think we were dealing with the `auth` proto object.